experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/lazyField.spec.ts

```diff
@@ -65,3 +92,316 @@ describe("lazyField", () => {
 		);
 	});
 });
+
+function createSingleValueTree<Kind extends FieldKind, Types extends AllowedTypes>(
+	builder: SchemaBuilder,
```

Comment:
schemaBuilder is a single muse mutable builder object. Passing it into things doesn't really make sense. It would make much more sense to pass in a TypedSchemaCollection. Since you also have the content for the tree, it seems like you want the "TreeContent" type which provides a way to pair schema with tree data for initialization purposes.

This is actually the type you are passing to forestWithContent already, so using it as input here would simplify things.

Actually, can you just move contextWithContentReadonly into utils and call that.